src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -329,29 +327,29 @@ private String getSubmissionFormTableBodyHtml(String questionNumberString, Strin
     private String getSubmissionFormMobileHtml(String questionNumberString, String responseNumberString,
             boolean sessionIsOpen, boolean isExistingResponse, FeedbackRubricResponseDetails frd) {
         StringBuilder mobileHtml = new StringBuilder();
-        String mobilePanelTemplate = FeedbackQuestionFormTemplates.RUBRIC_SUBMISSION_FORM_MOBILE_PANEL;
-        String mobilePanelFragmentTemplate = FeedbackQuestionFormTemplates.RUBRIC_SUBMISSION_FORM_MOBILE_PANEL_FRAGMENT;
+        String mobilePanelTemplate = FormTemplates.RUBRIC_SUBMISSION_FORM_MOBILE_PANEL;
+        String mobilePanelFragmentTemplate = FormTemplates.RUBRIC_SUBMISSION_FORM_MOBILE_PANEL_FRAGMENT;
 
         for (int i = 0; i < numOfRubricSubQuestions; i++) {
             StringBuilder panelBody = new StringBuilder();
             for (int j = 0; j < numOfRubricChoices; j++) {
                 String panelBodyFragment = Templates.populateTemplate(mobilePanelFragmentTemplate,
-                        "${qnIndex}", questionNumberString,
-                        "${respIndex}", responseNumberString,
-                        "${col}", Integer.toString(j),
-                        "${row}", Integer.toString(i),
-                        "${disabled}", sessionIsOpen ? "" : "disabled",
-                        "${description}", Sanitizer.sanitizeForHtml(this.getDescription(i, j)),
+                        Slots.QUESTION_INDEX, questionNumberString,
+                        Slots.RESPONSE_INDEX, responseNumberString,
+                        Slots.COL, Integer.toString(j),
+                        Slots.ROW, Integer.toString(i),
+                        Slots.DISABLED, sessionIsOpen ? "" : "disabled",
+                        Slots.DESCRIPTION, Sanitizer.sanitizeForHtml(this.getDescription(i, j)),
+                        Slots.CHECKED, isExistingResponse && frd.getAnswer(i) == j ? "checked" : "",
                         // Check if existing choice for sub-question == current choice
```

Comment:
This comment belongs to the `Slots.CHECKED` above, let's shift it to before that line like how it's done before.